src/xrpld/app/tx/detail/InvariantCheck.cpp

```diff
@@ -86,6 +87,8 @@ XRPNotCreated::visitEntry(
     std::shared_ptr<SLE const> const& before,
     std::shared_ptr<SLE const> const& after)
 {
+    std::stringstream ss;
+    ss << "XRPNotCreated::visitEntry start drops: " << drops_ << ". ";
```

Comment:
Are these debug checks that should be removed?

Reply:
fixed